src/components/renderers/layouts/masterdetail/masterdetail-directives.ts

```diff
@@ -1,125 +1,159 @@
 
-
-    restrict = "E";
+    restrict = 'E';
     replace = true;
     scope = {
         properties: '=',
         instance: '=',
-        select:'&'
+        select: '&'
     };
-    template = `
-        <div>
+    template = `<div>
             <accordion close-others="false">
-                <accordion-group is-open="status_attribute.open" ng-repeat="(key, value) in filter(properties)" ng-class="{'jsf-masterdetail-empty':isEmptyInstance(key)}">
+                <accordion-group is-open="status_attribute.open" 
+                                 ng-repeat="(key, value) in filter(properties)" 
+                                 ng-class="{'jsf-masterdetail-empty':isEmptyInstance(key)}">
                     <accordion-heading class="jsf-masterdetail-header">
-                        <span class="jsf-masterdetail-property">{{key}}</span> <i class="pull-right glyphicon" ng-class="{'glyphicon-chevron-down': status_attribute.open, 'glyphicon-chevron-right': !status_attribute.open}" ng-show="!isEmptyInstance(key)"></i>
+                        <span class="jsf-masterdetail-property">{{key}}</span> 
+                        <i class="pull-right glyphicon" 
+                           ng-class="{
+                              'glyphicon-chevron-down': status_attribute.open, 
+                              'glyphicon-chevron-right': !status_attribute.open
+                           }" 
+                           ng-show="!isEmptyInstance(key)">
+                        </i>
                     </accordion-heading>
 
-                    <accordion close-others="false" ng-show="!isEmptyInstance(key)">
-                        <accordion-group is-open="status_object.open" ng-repeat="child in instance[key]" ng-class="{'jsf-masterdetail-empty':!hasKeys(value.items)}">
+                    <accordion close-others="false" 
+                               ng-show="!isEmptyInstance(key)">
+                        <accordion-group is-open="status_object.open" 
+                                         ng-repeat="child in instance[key]" 
+                                         ng-class="{'
+                                           jsf-masterdetail-empty':!hasKeys(value.items)
+                                         }">
                             <accordion-heading>
-                                <span ng-click="selectElement(child,value)" ng-class="{'jsf-masterdetail-selected':selectedChild==child}">{{child.name!=undefined?child.name:child}}<!-- label provider needed --></span>
-                                <i class="pull-right glyphicon" ng-class="{'glyphicon-chevron-down': status_object.open, 'glyphicon-chevron-right': !status_object.open}" ng-if="hasKeys(value.items)"></i>
+                                <span ng-click="selectElement(child,value)" 
+                                      ng-class="{
+                                         'jsf-masterdetail-selected':selectedChild==child
```

Comment:
the current max line length is set to 100, so I had to come up with something ;)